core/sql/cli/Cli.cpp

```diff
@@ -10121,6 +10122,23 @@ Lng32 SQLCLI_LOBcliInterface
 	    if (inoutDescSyskey)
 	      *inoutDescSyskey = inDescSyskey;
 	  }
+        else
+          {
+            if (cliRC == 100)
+              {
+                if (dataLen)
+                  *dataLen = 0;
+                if (dataOffset)
+                  *dataOffset = 0;
+                if (blackBoxLen)
+                  *blackBoxLen = 0;
+                if (inoutDescPartnKey)
+                  *inoutDescPartnKey = descPartnKey;
+
+                if (inoutDescSyskey)
+                  *inoutDescSyskey = inDescSyskey;
+              }
+          }
```

Comment:
Should the two commented lines after this be deleted now (since you're handling the cliRC == 100 case)?